jquery.highchartTable.js

```diff
@@ -315,7 +315,8 @@
       var lineShadow  = $table.data('graph-line-shadow');
       var lineWidth   = $table.data('graph-line-width') || 2;
 
-      for(var i=0; i<9; i++) {
+      var nbOfColors = Math.max(defaultColors.length, themeColors.length);
+      for(var i=0; i < nbOfColors; i++) {
```

Comment:
Marrant, pour moi l'usage JS c'est justement de faire la déclaration du `i` dans le `for`. ^^
Ça doit dépendre de la religion de l'auteur.